src/components/footer/styledFooter.js

```diff
@@ -0,0 +1,69 @@
+import styled from "styled-components";
+
+export const FooterWrapper = styled.footer`
+    display: flex;
+    flex-direction: column;
+    background: #1C383C;
+    padding: 80px 0 60px 0;
+`
+export const FooterBlocks = styled.div` 
+    display: flex;
+    width: 88.89%;
+    justify-content: space-between;
+    margin: auto;
+`
+export const Block = styled.div`
+    display: flex;
+    flex-direction: column;
+`
+export const BlockTitle = styled.h3`
+    font-style: normal;
+    font-weight: normal;
+    font-size: 44px;
+    line-height: 52px;
+    color: #FFFFFF;
+    margin-bottom: 40px;
+`
+export const TextBlock = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 40px;
+`
+export const TextLine = styled.p`
+    font-style: normal;
+    font-weight: normal;
+    font-size: ${props => props.$size ? '14px' : '18px'};
+    line-height: ${props => props.$size ? '17px' : '21px'};
+    color: #FFFFFF;
+    margin-bottom: ${props => props.margin ? '10px' : null};
+    margin-right: ${props => props.$right ? '40px' : null};
+    white-space: pre-line;
+`
+export const LinkWrapper = styled.div`
```

Comment:
the text should be aligned in the middle of the image, try adding 'align-items: center;'. I think that will do the trick